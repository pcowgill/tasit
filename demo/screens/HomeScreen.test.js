import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import HomeScreen from "./HomeScreen";

describe("HomeScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    expect(shallow(<HomeScreen />)).toMatchSnapshot();
  });
});
