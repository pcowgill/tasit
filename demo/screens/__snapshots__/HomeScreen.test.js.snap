// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders the component 1`] = `
<Home
  onPress={[Function]}
/>
`;
